include/benchmark/benchmark.h

```diff
@@ -1212,7 +1213,7 @@ class BenchmarkReporter {
     bool error_occurred;
     std::string error_message;
 
-    int64_t iterations;
+    int iterations;
```

Comment:
What's this change?

Reply:
I figured the iteration count wouldn't go past 4 billion. It reverts the previous change, which is admittedly annoying.